inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -356,25 +366,36 @@ export class CanvasGraphService {
                 const timestamp = this.datasets.data.at(this.datasets.startingIndices.at(pointIndex));
                 const value = this.datasets.data.at(valueIndex);
 
+                if (!prevValue) {

```

Comment:
can it be 0? is 0 a valid value?

Reply:
It is a valid value, thanks for catching that!

Reply:
Updated!